hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/RDBTable.java

```diff
@@ -119,6 +119,16 @@ public boolean isEmpty() throws IOException {
     }
   }
 
+  @Override
+  public boolean isExist(byte[] key) throws IOException {
+    try {
+      return db.get(handle, key) != null;
+    } catch (RocksDBException e) {
+      throw toIOException(
+          "Failed to check if given key exists", e);
```

Comment:
This error message is kind of confusing. Perhaps replace it with, "Error in accessing  DB.".

Reply:
Addressed